### ISSUE TITLE
Catch up with sckit-learn 1.0

### ISSUE THREAD

**Issue by ablaom** [alan-turing-institute/MLJScikitLearnInterface.jl#40]
At time of writing, both SckitLearn.jl and MLJScikitLearnInterface.jl  pass tests with the latest version of sk-learn:

package                    |            build
---------------------------|-----------------
    joblib-1.1.0               |     pyhd8ed1ab_0         210 KB  conda-forge
    scikit-learn-1.0.2         |   py39h4dfa638_0        26.8 MB  conda-forge
    scipy-1.8.0                |   py39hee8e79c_1        25.4 MB  conda-forge
    threadpoolctl-3.1.0        |     pyh8a188c0_0          18 KB  conda-forge
 ------------------------------------------------------------

But a there are a ton of warnings about deprecations that will become permanent in scikit-learn 1.2. These need attention.

**Comment by ablaom** [comment]
For example, the `normalize` hyper-parameter in linear models needs deprecating. sk-learn now recommends a separate pre-processing step.

**Comment by ablaom** [comment]
Closed by #50
